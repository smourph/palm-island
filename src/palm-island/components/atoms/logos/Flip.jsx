import React from 'react';
import flip from '@internal/resources/images/logo-flip.png';
import './Logo.scss';

export const Flip = () => (
  <span className="logo logo-action logo-flip">
      <img src={flip} alt="flip"/>
  </span>
);

export default Flip;
