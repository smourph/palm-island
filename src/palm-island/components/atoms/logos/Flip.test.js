import React from 'react';
import {render, screen} from '@testing-library/react';
import Flip from './Flip';

it('should render snapshot', () => {
  const {asFragment} = render(<Flip/>);
  expect(asFragment().firstChild).toMatchSnapshot();
});

it('should render flip', () => {
  render(<Flip/>);

  const icon = screen.getByAltText('flip');
  expect(icon).toBeInTheDocument();
  expect(icon).toBeVisible();
  expect(icon).toHaveAttribute('src', 'logo-flip.png');
});
