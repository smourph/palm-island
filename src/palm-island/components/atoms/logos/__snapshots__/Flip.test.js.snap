// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render snapshot 1`] = `
<span
  class="logo logo-action logo-flip"
>
  <img
    alt="flip"
    src="logo-flip.png"
  />
</span>
`;
